### DISCUSSION TITLE
Unread notifications 'dot'

### DISCUSSION THREAD

**Discussion by catunderrain** [community/community#132420] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi! I have a small but annoying issue: my notifications on GitHub always have a dot. When I check, there are dots at Inbox and Participating, but inside, there's nothing! Could you explain what the issue is and how to turn off this dot when I literally have no notifications? Thank you.
![image](https://github.com/user-attachments/assets/c4be4b9a-298d-434a-a5b5-1bd0abc1e1f0)

**Comment by iamdulanga** [comment]
Hello, @catunderrain 

Try deleting cache on the your web browser.
If it doesn't work, your should make a report to github help.

Dulanga.

**Reply by catunderrain** [reply]
It's still not work bro. Thanks for your support, I appreciate it!

**Comment by ebndev** [comment, marked as answer]
Hi @catunderrain, thanks for flagging this!

I am keeping a log of this issue as other users have mentioned this as well. The GitHub product team will evaluate the feedback but will not be able to respond to every submission.

In the meantime, please see [this previous discussion](https://github.com/orgs/community/discussions/123129#discussioncomment-9451090) for possible solutions.

Feel free to ask questions on the original threads if you run into an issue.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Reply by catunderrain** [reply]
Thank you so much!
